starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function() {
 
     //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
+    $('#depositChecking').click(function() {
+      var curCheckBalance = $('#checkingBalance').text().replace('$','');
+      var convertCheck = parseFloat(curCheckBalance);
+      var inputC = parseFloat($('#amountChecking').val());
+      var finalCheckDep = convertCheck + inputC;
+      $('#checkingBalance').text('$' + finalCheckDep);
+    })
 
     //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
+    $('#withdrawChecking').click(function() {
+      var curCheckBalance = $('#checkingBalance').text().replace('$','');
+      var convertCheck = parseFloat(curCheckBalance);
+      var inputC = parseFloat($('#amountChecking').val());
+      var finalCheckWith = convertCheck - inputC;
+      if (inputC > convertCheck) {
+        return;
+      }
+        else {$('#checkingBalance').text('$' + finalCheckWith)};
```

Comment:
The `else` should have the same indentation as the `if`.

Also, you don't need the else in this case:
```
if (convertSav >= inputS) {
  $('#checkingBalance').text('$' + finalCheckWith)
}
```
This is cleaner than having the `return;` hanging out alone.